CL/cl_ext.h

```diff
@@ -253,6 +285,175 @@ typedef CL_API_ENTRY cl_command_queue
 #define CL_DEVICE_PREFERRED_CONSTANT_BUFFER_SIZE_AMD    0x4033
 #define CL_DEVICE_PCIE_ID_AMD                           0x4034
 
+typedef union
+{
+    struct { cl_uint type; cl_uint data[5]; } raw;
+    struct { cl_uint type; cl_uchar unused[17]; cl_uchar bus; cl_uchar device; cl_uchar function; } pcie;
+} cl_device_topology_amd;
```

Comment:
`cl_device_topology_amd` is part of `cl_amd_device_attribute_query` so it's not adding a new AMD extension, but I'd still rather see this added in #119 instead.